OsmAnd-java/src/main/java/net/osmand/search/SearchUICore.java

```diff
@@ -852,7 +854,36 @@ public boolean publish(SearchResult object) {
 			}
 			return false;
 		}
-		
+
+		public List<String> getTranslatedAdditionalInfo(Amenity amenity) {
+			List<String> result = new ArrayList<>();
```

Comment:
Old method used to exclude ```boolean excludeZipped``` now it doesn't filter anything. Implementation looks very strange it can't be reviewed without deep context understanding